src/test/java/org/broadinstitute/hellbender/utils/read/markduplicates/AbstractMarkDuplicatesCommandLineProgramTest.java

```diff
@@ -20,16 +20,22 @@
 
     protected abstract AbstractMarkDuplicatesTester getTester();
 
+  // ELIGIBLE_QUALITY is the minimum quality considered by the dataflow version
+  // for marking a specific pair as best. We use it to ensure that the expected
+  // pair wins the tie so that the tie is not broken randomly.
     protected final static int DEFAULT_BASE_QUALITY = 10;
+    protected final static int ELIGIBLE_QUALITY = 15;
```

Comment:
Oh, I didn't understand, but re-reading the note that makes sense.  I was thinking it just had to be higher to ensure that it was the best, not that it was meeting some threshold.  Sorry, reading failure on my part.